db/migrate/20221020182851_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly write the query like this so the foreign key name will be `author_id`.

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```